libursa/src/cl/mod.rs

```diff
@@ -779,7 +779,7 @@ pub struct BlindedCredentialSecretsCorrectnessProof {
 
 /// “Sub Proof Request” - input to create a Proof for a credential;
 /// Contains attributes to be revealed and predicates.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Deserialize)]
```

Comment:
Since you are not deriving `Serialize`, how would `SubProofRequest` be serialized in the first place?